recipe/meta.yaml

```diff
@@ -1,110 +1,104 @@
-{% set version = "1.1.0" %}
+{% set version = "1.6.0" %}
+{% set commit = "b31f58de6fa8bbda5353b3c77d9be4914399724d" %}
 
 package:
-  name: pytorch-cpu
+  name: pytorch-{{ pytorch_variant }}
   version: {{ version }}
 
 source:
-  # The tar source needs serious patching since it is trying to build
-  # all 3rd party modules
-  # url: https://github.com/pytorch/pytorch/archive/v{{ version }}.tar.gz
-  # fn: pytorch-v{{ version }}.tar.gz
-  # sha256: f360b7954e441bc5c4b36c49e711697a5c351de8b146c2982809385e3b3cfdcb
+  # for local testing use a tarball including submodules
   git_url: https://github.com/pytorch/pytorch.git
-  git_tag: v{{ version }}
-
-# The build needs to set many environment variables
-# This is an easy way to do it cross platform
-{% set export = "export" %}  # [not win]
-{% set export = "set" %}     # [win]
+  git_tag: {{ commit }}
+  patches:
+    - nccl_socket.patch
+    - remove_shared_linker_flag_override.patch
+    - fix_std_stdint.patch
+    - fix_dispatch_apply_auto.patch
+    - fix_map_anonymous.patch
+    - cpp_extension.patch
```

Comment:
Have these or similar patches made it upstream? If not, should we make PRs?

Reply:
I don't think so. These patches are from AnacondaRecipes. It'd be good to upstream at least some of the patches.